packages/zmarkdown/__tests__/sanitizer.tests.js

```diff
@@ -0,0 +1,28 @@
+const remarkConfig = require('../config/remark')
+const rebberConfig = require('../config/rebber')
+
+// Disable typographical corrections for easier testing
+remarkConfig.noTypography = true
+remarkConfig._test = true
+
+const zmarkdown = require('../server')
+
+const renderString = (config = {remarkConfig, rebberConfig}) =>
+  (input) =>
+    zmarkdown(config).renderString(input).then((vfile) =>
+      vfile.toString().trim())
+
+describe('#ping', () => {
+  it('do not oversanitive ping', () => {
+    const rendered = renderString()('@Clem')
+    expect(rendered).resolves.toContain('class="ping ping-link"')
```

Comment:
should be ok but I don't think class order is specified in HTML

Reply:
What do you mean "class order"? This test only checks for the two classes somewhere in the code, it does not check the order of the two classes, it is not the objective.

Can you run it removing the "l" from class; the test does not fail but it should.

Reply:
A lot of tests in the library rely on this ordering, either with `toContain` or `toEqual`.

Reply:
ok, so perfect :)